evm_loader/lib/src/account_storage.rs

```diff
@@ -114,6 +115,19 @@ pub struct EmulatorAccountStorage<'rpc, T: Rpc> {
     logs_stack: Vec<usize>,
 }
 
+async fn get_sysvar<T>(rpc: &dyn Rpc) -> NeonResult<T>
```

Comment:
You can use `rpc: &impl Rpc` here. I think the generated code will be simpler (no need to create a virtual function table for Rpc trait).